### PR TITLE
ISSUE-88 # multi load for production like load generation

### PR DESCRIPTION
**Pull request by authorjapps**

### CODE REVIEW

File: src/test/resources/load_config_test.properties [hunk 1]
```diff
@@ -2,15 +2,15 @@
 # to simulate a load test with 5 concurrent users then you need to enter 5 as the value for this property. A high end
 # machine will be able to spawn more number of threads. To keep the consistent(or nearly consistent) gap between the
 # threads, adjust this number with 'ramp.up.period.in.seconds' and the actual response time of the API end point.
```
Comment (nirmalchandra) [inline src/test/resources/load_config_test.properties]: Update this `property` how this affects multiple load

Comment (authorjapps) [reply src/test/resources/load_config_test.properties]: Updated this file as well as wiki page.

Comment (nirmalchandra) [reply src/test/resources/load_config_test.properties]: Worth creating a WikiPage for this multi-load feature

Comment (authorjapps) [reply src/test/resources/load_config_test.properties]: #91 has been raised for this.

**Pull request merged**